walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -0,0 +1,274 @@
+Additionally we going to explore `Mobile Developer Console` that is going to help us to orchiestrate all services and obtain mobile client configuration.
+
+Our demo showcase application will consist of:
+
+- Node.js server side application that will be deployed to OpenShift user namespace.
+Server side implements example TODO GraphQL API and connects to Postgress 
+
+- Ionic Framework and Apache Cordova client side application that we going to launch in CodeReady workspace
+
+As part of this walkthrough we going to utilize following Mobile Developer Services:
+
+- Data Sync Framework 
+Adds realtime data synchronization and offline capabilities to your mobile app
+
+- Identity Management 
+Adds authentication and authorization to your mobile app
+
+- Mobile Metrics
+Gather metrics on mobile apps, device versions, device security checks and back-end mobile service usage
+
+image::images/artifacts.png[integration, role="integr8ly-img-responsive"]
+
+[type=walkthroughResource, serviceName=openshift]
+.OpenShift Project
+****
+* link:{openshift-host}/console/project/{walkthrough-namespace}/overview[OpenShift Project, window="_blank"]
+****
+
+[type=walkthroughResource]
+.Mobile Developer Console
+****
+* link:{route-mdc-server-host}[Mobile Developer Console, window="_blank"]
+****
+
+[type=walkthroughResource]
+.Mobile Developer Services Documentation
+****
+* link:https://access.redhat.com/documentation/en-us/red_hat_mobile_developer_services/1[Documentation, window="_blank"]
+****
+
+[type=walkthroughResource,serviceName=codeready]
+.CodeReady Workspaces
+****
+* link:https://developers.redhat.com/products/codeready-workspaces/overview/[Overview, window="_blank"]
+* link:https://access.redhat.com/documentation/en-us/red_hat_codeready_workspaces_for_openshift/1.0.0/[Documentation, window="_blank"]
+****
+
+[type=walkthroughResource,serviceName=openshift]
+.Red Hat OpenShift
+****
+* link:{openshift-host}/console[Console, window="_blank"]
+* link:https://help.openshift.com/[Openshift Online Help Center, window="_blank"]
+* link:https://blog.openshift.com/[Openshift Blog, window="_blank"]
+****
+
+
+// NOTES: Till all services will be avaiable as operators inside openshift cluster we will require additional installation steps
+// Developers need to provision all services using Tech Preview. 
+// To do that please follow README in https://github.com/aerogear/mobile-services-installer
+
+[time=20]
+== Create DataSync Showcase Server Application from OpenShift template
+
+DataSync framework is an set of libaries that will allow developers to easily create mobile apps that can sync data to the their backend. The DataSync libaries are based on the popular Apollo GraphQL server, with the following additional functionalities:
+
+. Offline Support
+. Conflict Resolution
+. Easy integration with other mobile services
+
+This section describes how to deploy the example DataSync server in an Integreatly cluster from the OpenShift Service Catalog.
```

Comment:
Integreatly?
how about creating an attribute for this, so we can still support community